agent/acs/client/acs_client_test.go

```diff
@@ -1,4 +1,5 @@
-// Copyright 2014-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// +build unit
```

Comment:
> To distinguish build constraints from package documentation, a series of build constraints must be followed by a blank line.

I'm not sure if this is enforced? But we might want to change it.

https://golang.org/pkg/go/build/#hdr-Build_Constraints

Reply:
Looks like this is not enforced. But better to change this.